src/ckptserializer.h

```diff
@@ -32,10 +32,8 @@ namespace CkptSerializer
 {
 int openCkptFileToWrite(const string &path);
 void createCkptDir();
-void writeCkptImage(void *mtcpHdr,
-                    size_t mtcpHdrLen,
+void writeCkptImage(DmtcpCkptHeader ckptHdr,
```

Comment:
Let's add a comment above here.  Perhaps something like the following, but re-format it.
```
// ckpt file may be gzipped, and re read it from a pipe.  We can't lseek on a pipe.
// So, we write the DmtcpCkptHdr twice.  The first header is read by 'dmtcp_restart'.
// And then 'dmtcp_restart' execs into 'mtcp_restart'.  'mtcp_restart inherits the fd,
// and then reads the second copy of DmtcpCkptHdr.